### PR TITLE
docs: cli: add information about coder update command

### PR DESCRIPTION
**Pull request by johnstcn**

### CODE REVIEW

**Comment by johnstcn** [comment]
[ch16788]

**Comment by johnstcn** [comment]
[ch15422]

File: cli/installation.md [hunk 1]
```diff
@@ -55,3 +55,20 @@ Generate a static authentication token with the following command:
...
+coder update
+```
+
+The `coder update` command accepts the following arguments:
+
+- `--coder`: specify the Coder instance the CLI should use to query the version
+- `--version`: specify the version that the CLI should download and upgrade to
+- `--force`: omit prompts asking for change confirmations
+
+**Note:** Coder CLI will not update if it is under certain paths.
```
Comment (khorne3) [inline cli/installation.md:74]: @johnstcn can we expand on this note a bit? are there specific paths the user should watch out for? or, conversely, should the user make sure the cli stays in its default location?

Comment (johnstcn) [inline cli/installation.md:74]:
```suggestion
**Note:** Coder CLI will not update if it is located under `/var/tmp/coder` or `C:\Windows`.
```

Comment (johnstcn) [reply cli/installation.md]: Updated with specifics!

**Review by khorne3 — APPROVED** [review]

**Comment by vapurrmaid** [comment]
@all-contributors please add @johnstcn  for content

**Pull request merged**